Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/workflows/deploy08.yml

```diff
@@ -0,0 +1,52 @@
+name: 'Terraform'
```

Comment:
Please indicate the sections in the code that are only relevant for the B/G scenario

Reply:
fixed